project/apps/notify/notify.compose.dev.yml

```
@@ -0,0 +1,64 @@
+version: '3.9'
+services:
+  readme.notify.fakesmtp:
+    image: gessnerfl/fake-smtp-server:2.2.0
+    container_name: 'readme.notify.fakesmtp'
+    hostname: 'readme.notify.fakesmtp'
+    ports:
+      - '8025:8025'
+      - '1085:8080'
+
+  readme.notify.rabbitmq:
+    image: rabbitmq:3.11-management
+    container_name: readme.notify.rabbitmq
+
```

Хорошая практика добавлять

```suggestion
    hostname: 'rabbitmq'
```